pyDeltaRCM/deltaRCM_driver.py

```diff
@@ -5,7 +5,23 @@
 import datetime
 
 class pyDeltaRCM(Tools):
-    
+
+    _input_var_names = (
```

Comment:
I'd support removing all this stuff declarations from `pyDeltaRCM` and get it all into the BMI wrapper. I don't think we have to actually implement any of this here? Do these dicts and tuples get used internally?

Reply:
The problem is that these dictionaries are ]used to parse the input yaml file](https://github.com/DeltaRCM/pyDeltaRCM_WMT/blob/develop/pyDeltaRCM/init_tools.py#L42-L53).